end-to-end-tests/pageObjects/extensionConsole/modsPage.ts

```diff
@@ -158,8 +159,8 @@ export class ActivateModPage extends BasePageObject {
     await this.page.goto(this.activateModUrl);
 
     await expect(this.getByText("Activate Mod")).toBeVisible();
-    // Loading the mod details may take more than 5 seconds
-    await expect(this.getByText(this.modId)).toBeVisible({
+    // Loading the mod details may take a long time. Using ensureVisibility because the modId may be attached and hidden
```

Comment:
nit: I kind of liked the specificity of 5 seconds vs "a long time"
```suggestion
    // Loading the mod details could take several seconds. Using ensureVisibility because the modId may be attached and hidden
```